apps/client/app/managerpanel/page.tsx

```diff
@@ -0,0 +1,70 @@
+'use client'
```

Comment:
What is this page for? It looks the same as the normal dashboard.

If this is just for the future, could we please add a "Manager Dashboard" badge or something so that we can tell the difference?